submitted_models/cerberus_gagarin_sensor_config_1/README.md

```diff
@@ -0,0 +1,23 @@
+# darpa_virtual_model_submit
+Simulation models to submit to Darpa | CERBERUS Aerial Scouts (Alpha/Bravo/Charlie)
+
+## workspace setup
+Follow these step to setup the Darpa Subt Virtual Environment: 
+
+https://bitbucket.org/osrf/subt/wiki/tutorials/SystemSetupInstall
+
+## add a new model
+
+```
+cd ~/subt_ws/src/subt/submitted_models
+
+git clone https://github.com/unr-arl/darpa_virtual_model_submit.git cerberus_gagarin_sensor_config_1
+
+cd ~/subt_ws/
+catkin_make install
+source install/setup.bash
+```
+
+## launch the simulation
+ign launch -v 4 src/subt/submitted_models/cerberus_gagarin_sensor_config_1/launch/gagarin_test.ign robotName:=gagarin modelName:=CERBERUS_GAGARIN_SENSOR_CONFIG_1 localModel:=true
```

Comment:
I tried running this cmd with the submitted model in this PR but I don't see the robot in the world. It also looks like `modelName` is not used. 

On the other hand, the instructions in `specifications.md` works for me

Reply:
I believe this method was active since the time the repo was on bitbucket. I think we carried it over from there. I'll change it to be the same as specifications.md

This method is mentioned to launch the robots in all the example.ign files, however, it does not work anymore for me( I think it worked on my device around the time when subt repo was on bitbucket).
```
<?xml version="1.0"?>
<!-- Usage: ign launch path/to/example.ign robotName:=<X1>

   Parameters:
    robotName: Name to be assigned to model
-->
```